Makefile

```diff
@@ -131,7 +131,7 @@ install-protoc:
 
 PROTOCGENGO=$(shell pwd)/buildtools/protoc-gen-go
 protos: install-protoc clean-protos
-	./scripts/protoc_gen_plugin.bash \
+	PATH=$(PROTOCGENGO):$(PATH) ./scripts/protoc_gen_plugin.bash \
```

Comment:
This wasn't using the `protoc-gen-go` inside the `buildtools` folder. This makes sure it does and results in a newer version of `protoc-gen-go` being used than on @sanderpick's computer, so all the generated code shows updates in this PR.

Reply:
Glad to see that 9k of code changes are related to this for review :)